cocos/platform/android/java/src/org/cocos2dx/lib/Cocos2dxGLSurfaceView.java

```diff
@@ -199,7 +199,7 @@ public void run() {
             }
         });
         this.setRenderMode(RENDERMODE_WHEN_DIRTY);
-        //super.onPause();
```

Comment:
The meaning of this PR is finger out clearly that we want to preserve GLContext onPause, the older workaround is this comments.